src/rest_api/classes/variation/widgets/molecular_details.clj

```diff
@@ -681,6 +681,78 @@
                                    {:item
                                     (pack-obj t)
 
+                                    :vep_consequence
+                                    (some-> (:molecular-change/vep-consequence h)
+                                            (first)
+                                            (str/replace "_" " "))
```

Comment:
I think this might be an error if the stirring is nil. I am not sure if you need to do a check first. same with the rest of the fields added in this pull request. If not everything looks great

Reply:
Thanks for checking! `some->` should short circuit the thread if `nil` is returned by any step